packages/yourloops/lib/utils.ts

```diff
@@ -103,6 +103,13 @@ export function getUserLastName(user: IUser): string {
   return user.profile?.lastName ?? user.profile?.fullName ?? user.username;
 }
 
+/**
+ * Return the user full name
+ */
+export function getUserFullName(user: IUser): string {
+  return user.profile?.fullName ?? "";
```

Comment:
Maybe the email can be used if no fullName?
Having an empty field is not very appealing.
```suggestion
  return user.profile?.fullName ?? getUserEmail(user);
```

Reply:
When I created this function, I was actually planning on using the one that created Arnaud in his ongoing PR. This PR still being a wip, I still don't know if I'm going to use this function :wink: